### PR TITLE
2021-2022 fall update

### PR DESCRIPTION
**Pull request by DavidPurdy1**
fixed website for mobile devices, updated pictures/info/hares, refactored a little bit.

In the future would like to get rid of the embedded calendar which has a lot of dependencies and replace it with easy to change image of the rush calendar because it never gets updated otherwise. Could also add links to the rush page like the linktree link we have. Also not familiar with materialize.css, but why do we have 2 of them in the project?

### CODE REVIEW

**Comment by schwalbs** [comment]
> refactored a little bit

> touches 97 files

**Comment by schwalbs** [comment]
> Also not familiar with materialize.css, but why do we have 2 of them in the project?

I don't remember, this was my first web project I created from scratch back in 2015 I think. That CSS library is way outdated. One is the minified I believe and the other is the full file.

File: js/site-components.js
Comment (schwalbs) [inline js/site-components.js:5]: Nice

File: index.html
Comment (schwalbs) [inline index.html]: Do we have that new signature?

File: index.html
Comment (schwalbs) [inline index.html]: Pretty sure he's not the president anymore

File: history.html
Comment (schwalbs) [inline history.html]: We should do this with a CSS class

```css
@media all and (min-width: 480px){
  .hidden-tablet-up {
    display: none;
  }
}
```

**Review by schwalbs — CHANGES_REQUESTED** [review]

**Comment by DavidPurdy1** [comment]
> > refactored a little bit
> 
> > touches 97 files

haha yeah she needed some work

**Comment by yorker78** [comment]
I'm afk till this weekend, I'll take a look then. Don't feel like you NEED approval from us, but if you want other eyes on it, I'll be happy to once I'm able.

**Comment by DavidPurdy1** [comment]
Okay, cool thanks. Didn't want to just come in a change a whole bunch of things on my first PR here. The last thing I am going to do is probably change the rush calendar to just be jpg instead of the interactive calendar. Once I finish that I might merge it in.

**Comment by yorker78** [comment]
If you're webmaster, then you're the primary maintainer and can do as you please (just don't fuck it up too much), otherwise they're the person to get approval from. The alumni are just here as an extra set of eyes and experience to reach out to for assistance. I'm not trying to downplay our contributions, but at the end of the day it's a website by & for actives. Also if you have an idea and you're unsure how to implement, you can try creating an issue and assigning it (no guarantee it'll actually get picked up by the person you've assigned it to).

File: brothers.html
Comment (schwalbs) [inline brothers.html:11]: We're yeeting `css/materialize.min.css`, does it get auto-generated?

**Comment by schwalbs** [comment]
Like Billy said, go bonkers in this repo, it's all yours. I won't feel bad if you make my crappy code better. Go for a full rewrite if you want, there's a lot that can be improved on here. More templating and a newer CSS framework would really improve it.

I'm fairly active on Github for work and would be happy to review code or help with changes.

File: brothers.html
Comment (cocicat) [reply brothers.html:11]: It seems like its not getting yeeted, but rather condensed all into line six for viewing sake. Is this accurate @DavidPurdy1 ?

Comment (schwalbs) [reply brothers.html:11]: oh you right. I assumed 7500 lines of deletion was a full yeet.

**Review by cocicat — APPROVED** [review]
Fantastic changes, especially replacing the site-header and site-footer. Thanks David.

**Pull request merged**
